beekeeper-cleanup/src/test/java/com/expediagroup/beekeeper/cleanup/aws/S3ClientTest.java

```diff
@@ -207,6 +207,21 @@ void deleteObjectsInDirectory() {
     assertThat(amazonS3.doesObjectExist(bucket, key2)).isFalse();
   }
 
+  @Test
```

Comment:
You should also add a test to see what happens when one of the delete operations fails.

Reply:
will work on this on the new ticket